instance/instance_external_test.go

```diff
@@ -339,6 +343,7 @@ func TestUpdateInstanceReturnsOk(t *testing.T) {
 		So(len(mockedDataStore.UpsertEditionCalls()), ShouldEqual, 1)
 		So(len(mockedDataStore.GetNextVersionCalls()), ShouldEqual, 1)
 		So(len(mockedDataStore.UpdateInstanceCalls()), ShouldEqual, 1)
+		So(len(mockedDataStore.AddVersionDetailsToInstanceCalls()), ShouldEqual, 1)
```

Comment:
Consider adding assertions to other scenarios to check that your new function function is not called: `So(len(mockedDataStore.AddVersionDetailsToInstanceCalls()), ShouldEqual, 0)`